### DISCUSSION TITLE
Help required: We are trying to build and host WIKI.JS on Windows Server

### DISCUSSION THREAD

**Discussion by Kamalsurya** [requarks/wiki#3441] [Help / Questions]
**Question**
Hi, We are trying to build the application on Windows server. Having trouble to build and host on IIS. Can i get a build content. I have the database scripts so that i can insert the ConnectionString.
Can you help me out.

**Host Info (please complete the following information):**
OS: Windows Server 2016 
Wiki.js version: 2.0.1
Database engine: MS SQL Server 2017

**Comment by NGPixel** [comment]
https://github.com/Requarks/wiki/releases/download/2.5.144/wiki-js-windows.tar.gz
